sources/course_examples.md

```diff
@@ -0,0 +1,5 @@
+# Examples of Course Curricula
+
+[**NOTE**: Anyone have any suggestions of items to add here?]
+The following are examples of curricula for course on C++:
+...
```

Comment:
@jcvw What should we do with this section? Keep it, and leave empty with node for future work, or remove?

Reply:
I think we can remove this.